server/mongo/models/Ticket.ts

```diff
@@ -4,6 +4,8 @@ interface TicketMongoType {
   _id: Types.ObjectId
   userId: Types.ObjectId
   eventId: Types.ObjectId
+  isFirstHand: Boolean,
```

Comment:
I think that isSecond hand is better, because the field is used for the second hand